extensions/eclipse-che-theia-plugin/src/che-proposed.d.ts

```diff
@@ -15,6 +15,34 @@
 
 import { che as cheApi } from '@eclipse-che/api'
 import * as theia from '@theia/plugin';
+import {
+    CompletionContext,
+    CompletionResultDto,
+    SignatureHelp,
+    Hover,
+    DocumentHighlight,
+    Range,
+    TextEdit,
+    FormattingOptions,
+    Definition,
+    DocumentLink,
+    CodeLensSymbol,
+    DocumentSymbol,
+    ReferenceContext,
+    Location,
+    SignatureHelpContext,
+    CodeActionContext,
+    CodeAction,
+    FoldingRange,
+} from '@theia/plugin-ext/lib/common/plugin-api-rpc-model';
+import { UriComponents } from '@theia/plugin-ext/lib/common/uri-components';
+import { SymbolInformation } from 'vscode-languageserver-types';
```

Comment:
`@theia/plugin` for all these one as well

Reply:
it would be nice if here, we could keep only from @theia/plugin. For the other locations, it's not important as it's implementation details.

It's because by doing this, we ensure that the dependencies are very small when linking to `@eclipse-che/plugin`